### DISCUSSION TITLE
ImGui window makes the screen appear as background colour

### DISCUSSION THREAD

**Discussion by jnmaloney** [google/filament#3377] [General]
https://user-images.githubusercontent.com/5510983/102577672-32c0a680-4144-11eb-9635-bca55d089ece.mov

I am trying to use Filament as a renderer with an independent ImGui intance (not using Filament materials), and sharing the same GL context in WebGL.

Not sure what I'm doing wrong but there is something like clipping in the screen rectangle. As the window moves, the Filament scene can become visible or invisible .

https://github.com/jnmaloney/Minimal_Filament_ImGui

**Comment by romainguy** [comment]
Filament does its own GL state tracking and can't be mixed arbitrarily with other renderers (and it's probably true the other way around). We have a filagui library to use ImGui in Filament though.
